app/src/main/res/layout/custom_dialog.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="16dp"
+        android:textSize="16sp" />
+
+    <ListView
+        android:id="@+id/listView"
```

Comment:
Please use more descriptive resource ID names. Instead of stating what it _is_ (TextView, ListView, Button), it's easier to differentiate between ID names (especially when they show up as Android Studio autocomplete suggestions) if the IDs describe _how_ they're used (`phone_number_selection_dialog_message`, `phone_number_selection_dialog_list` etc)